### PR TITLE
Replace active and ready with status in workflow

### PR DESCRIPTION
**Pull request by Beacze**
workflow
workflow2

### CODE REVIEW

File: Kitodo/src/main/resources/messages/messages_de.properties [hunk 1]
```diff
@@ -27,7 +27,7 @@ active=Aktiv
 activeUsers=Aktive Benutzer
 addDemoTask=Beispielaufgabe hinzuf\u00FCgen
 addFilter=Filter hinzuf\u00FCgen
-addMessage=Nachricht hinzuf\u00FCgen
+addMessage=Nachricht hinzuf\u00FCgens
```
Comment (Kathrin-Huber) [inline Kitodo/src/main/resources/messages/messages_de.properties]: Why did you change it to "hinzufügens"?

Comment (Beacze) [reply Kitodo/src/main/resources/messages/messages_de.properties]: Mistake, now it is removed.
